plugin/storage/cassandra/spanstore/reader.go

```diff
@@ -289,6 +289,7 @@ func (s *SpanReader) queryByTagsAndLogs(tq *spanstore.TraceQueryParameters) (dbm
 			model.TimeAsEpochMicroseconds(tq.StartTimeMax),
 			tq.NumTraces,
 		)
+		query.PageSize(0)
```

Comment:
1. why not in `s.executeQuery()`?
 1. this needs to be `query = query.PageSize(0)`